### ISSUE TITLE
Vesicoureteral reflux grade II

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6826]
**Preferred term label:**
Vesicoureteral reflux grade II

**Synonyms**
VUR grade II, VUR II

**Definition (free text, please give PubMed ID)**
Vesicoureteral reflux into a non-dilated pyelocalyceal system.

PMID: 3975102, 18322164

**Parent term (use hpo.jax.org/app)**
Low-grade vesicoureteral reflux (newly proposed term)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
